### ISSUE TITLE
解析图片后没有输出有效的结果

### ISSUE THREAD

**Issue by penond** [deepseek-ai/DeepSeek-OCR#29]
解析指定图片，执行python run_dpsk_ocr.py后 没有输出任何有效的结果

  reulst.mmd  里只有一个图片标记：  idden & gt;![] (images/0.jpg)
  而 result_with_boxes.jpg 就是输出的原图

过程如下：
You are using a model of type deepseek_vl_v2 to instantiate a model of type DeepseekOCR. This is not supported for all configurations of models and can yield errors.
You are attempting to use Flash Attention 2.0 without specifying a torch dtype. This might lead to unexpected behaviour
You are attempting to use Flash Attention 2.0 with a model not initialized on GPU. Make sure to move the model to GPU after initializing it on CPU with `model.to('cuda')`.
Some weights of DeepseekOCRForCausalLM were not initialized from the model checkpoint at /data/models/DeepSeek-OCR and are newly initialized: ['model.vision_model.embeddings.position_ids']
You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.
/root/miniconda3/envs/deep-ocr/lib/python3.11/site-packages/transformers/generation/configuration_utils.py:590: UserWarning: `do_sample` is set to `False`. However, `temperature` is set to `0.0` -- this flag is only used in sample-based generation modes. You should set `do_sample=True` or unset `temperature`.
  warnings.warn(
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:None for open-end generation.
The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
The `seen_tokens` attribute is deprecated and will be removed in v4.41. Use the `cache_position` model input instead.
`get_max_cache()` is deprecated for all Cache classes. Use `get_max_cache_shape()` instead. Calling `get_max_cache()` will raise error from v4.48

BASE:  torch.Size([1, 256, 1280])
PATCHES:  torch.Size([2, 100, 1280])

The attention layers in this model are transitioning from computing the RoPE embeddings internally through `position_ids` (2D tensor with the indexes of the tokens), to using externally computed `position_embeddings` (Tuple of tensors, containing cos and sin). In v4.46 `position_ids` will be removed and `position_embeddings` will be mandatory.
idden&gt;<|ref|>image<|/ref|><|det|>[[0, 0, 999, 999]]<|/det|>

image size:  (729, 375)
valid image tokens:  331
output texts tokens (valid):  21
compression ratio:  0.06

===============save results:===============
image: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 1/1 [00:00<00:00, 35544.95it/s]
other: 0it [00:00, ?it/s]

**Comment by ZiyanZhu1994** [comment]
请问是怎么解决的？谢谢！
